front-page.php

```diff
@@ -34,11 +34,12 @@
 
 		</main><!-- #main -->
 	</div><!-- #primary -->
-<?php get_template_part( 'template-parts/content', 'secondary' ) ?>	
+<?php get_template_part( 'template-parts/content', 'secondary' ) ?>
 <?php get_sidebar(); ?>
 </div><!-- .o-layout -->
 
 <?php get_template_part( 'template-parts/acf', 'links' );?>
+<?php get_template_part( 'template-parts/acf', 'cta-cards' );?>
```

Comment:
If no CTA Cards are added, is any empty markup output here?

Reply:
I reckon we're ok as this line checks for cards before outputting any markup:
`if( have_rows('cards') ):`